src/app/shared/models/shelf-configuration.ts

```diff
@@ -0,0 +1,12 @@
+import { Tv2ActionContentType } from './tv2-action'
+
+export interface ShelfConfiguration {
+  id: string
+  actionPanels: ShelfActionPanelConfiguration[]
```

Comment:
For the explicitness I would prefer this to be `actionPanelConfigurations`.

Reply:
Updated